tests/run-make/msvc-opt-minsize/rmake.rs

```diff
@@ -0,0 +1,19 @@
+// A previously outdated version of LLVM caused compilation failures on Windows
+// specifically with optimization level `z`. After the update to a more recent LLVM
+// version, this test checks that compilation and execution both succeed.
+// See https://github.com/rust-lang/rust/issues/45034
+
+// NOTE: This isn't an UI test despite its simplicity, because UI tests
+// cannot contain opt-level modifications.
```

Comment:
I'm not so sure about that. It's what was said [here](https://github.com/rust-lang/rust/pull/45824#issuecomment-343591668), but that was 7 years ago and I definitely see some instances of opt-level in the UI testsuite. Let me know if you think I should attempt a conversion to an UI test.

Reply:
Yeah this really looks like it should be a UI test, they can set `-Copt-level` flags now.